src/content-script.ts

```diff
@@ -4,14 +4,47 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+const regexPathName = /\/.+\/.+\/(issues|pull)\/\d+/;
+const replySummaryId = "saved-reply-new_comment_field";
+const footprint = "refined-saved-replies";
+
+// hack for detecting page change
```

Comment:
[Praise] This isn't a hack, it's a very legitimate and reasonable pattern - the right thing to use for this kind of situation! 😄 

```suggestion
```